interface_config.js

```diff
@@ -4,6 +4,7 @@ var interfaceConfig = {
     // TO FIX: this needs to be handled from SASS variables. There are some
     // methods allowing to use variables both in css and js.
     DEFAULT_BACKGROUND: '#474747',
+    DEFAULT_LOGO_URL: '../images/watermark.png',
```

Comment:
Just double-checking: this works ok when accessing the URL from a tenant right?

Reply:
Right. This will be displayed for all tenant urls that do not have a logo in `dynamic-branding`.